libpermian/plugins/__main__.py

```diff
@@ -0,0 +1,34 @@
+import argparse
+from libpermian import plugins
+
+
+def list_modules(names=False, paths=False):
+    paths_list = []
+    names_list = []
+    plugins.load()
+    plugins.loaded_plugin_modules()
+    for p in plugins.loaded_plugin_modules():
+        paths_list.append(p.__path__[0])
+        names_list.append(p.__name__)
+    if paths and not names:
+        print('\n'.join(paths_list))
+    elif names and not paths:
+        print('\n'.join(names_list))
+    else:
+        print('\n'.join([ f'{n}:{p}' for n, p in zip(names_list, paths_list) ]))
+
+
+parser = argparse.ArgumentParser()
+subparsers = parser.add_subparsers(dest='command')
+parser_list = subparsers.add_parser('list',
+                            help='Lists all enabled permian plugins')
+parser_list.add_argument('--paths', action='store_true',
+                            help='Lists paths of all enabled permian plugins')
+parser_list.add_argument('--names', action='store_true',
+                            help='Lists names of all enabled permian plugins')
+args = parser.parse_args()
+
+if args.command == 'list':
+    list_modules(names=args.names, paths=args.paths)
+else:
+    parser.print_help()
```

Comment:
This is handled directly by `ArgumentParser`, when invalid command is provided, the help is printed and proper return code is set.
I'm also thinking that "list" should be default command but I'm not sure how this could be done.